### PR TITLE
Add a website document config file of v8.3.0

### PR DESCRIPTION
**Pull request by Jtrust**
- [x] Add a website document sidebar config file of v8.3.0

### CODE REVIEW

**Review by wu-sheng — COMMENTED** [review]
Please don't update submodules.

**Comment by Jtrust** [comment]
ok

File: docs/main_8_3_0.yml [hunk 1]
```diff
@@ -0,0 +1,164 @@
+name: "v8.3.0"
+children:
+  - name: "Welcome"
+    path: "/docs/main/v8.3.0/readme"
```
Comment (wu-sheng) [inline docs/main_8_3_0.yml:4]: Could `/docs/main/v8.3.0` be auto generated? Or use one config at the top?

**Comment by wu-sheng** [comment]
Take a reference here, https://github.com/apache/skywalking/pull/6366#pullrequestreview-588715124

**Comment by Jtrust** [comment]
> Take a reference here, [#6366 (review)](https://github.com/apache/skywalking/pull/6366#pullrequestreview-588715124)

👌
Happy New Year!

**Comment by wu-sheng** [comment]
> > Take a reference here, [#6366 (review)](https://github.com/apache/skywalking/pull/6366#pullrequestreview-588715124)
> 
> 👌
> Happy New Year!

You too, happy new year!

**Pull request merged**